src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -307,8 +323,9 @@ export default {
 			flex-basis: calc(100% / 8);
 			text-align: center;
 
-			&:hover::before {
-				background-color: var(--color-background-hover);
+			&:hover,
+			&.emoji-mart-emoji-selected{
+				border: 1px solid var(--color-primary);
```

Comment:
Border takes space, not the outline :)

Reply:
But I just tested and it doesn't seems to make the layout jump :thinking: 
I guess that's alright, even if I would prefer being consistent with the categories styling